resources/definitions/tronxy_x5sa-pro.def.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": 2,
+    "name": "Tronxy x5sa",
```

Comment:
This one should have some distinguishing name. I think you meant `Tronxy x5sa Pro`?

Reply:
Maybe you want to give it a consistent capitalisation as well. Normally we ignore "stylised" capitalisation because that's just marketing tricks to make it catch the attention, but you can consider using `Tronxy X5SA Pro` here.

Reply:
Hmmm, ok. I changed "machine name" to lower case letters, "name" to upercase. Please tell me, if this is not OK.

Reply:
Better now. The main problem is that there were two printers with the same name.